### DISCUSSION TITLE
Better edit controls in the mobile version

### DISCUSSION THREAD

**Discussion by EBatata** [ppy/osu#19632] [Ideas] [answered]
I was trying to edit a map in the mobile version but it's impossible to delete anything since all you have is a mouse that can only left click. I think there should be actual commands for the edit area like double clicking or triple clicking being the right click, or add buttons made only for those purposes.

![Screenshot_20220807-174945](https://user-images.githubusercontent.com/110789105/183310782-81e6f22d-f83e-41ec-8429-c90da73b27f3.jpg)

Help.

**Comment by peppy** [comment, marked as answer]
Home for right click support will arrive soon, but we will also eventually improve controls specifically with mobile in mind.
